app/views/docs/functions.phtml

```diff
@@ -347,50 +440,6 @@ $image = new View(__DIR__.'/../general/image.phtml');
         ->render();
 ?>
 
-<h3><a href="/docs/functions#runOffline" id="runOffline">Run Offline</a></h3>
-
-<p>You can quickly run your Cloud Function locally by executing the Appwrite runtime locally using its corresponding Docker image. You can pass your offline container any environment variables your script needs and your entry command like in the example below.</p>
-
-<ul class="phases clear" data-ui-phases>
-    <li>
-        <h3>Unix</h3>
-        <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-            <pre class="line-numbers"><code class="prism language-bash" data-prism>docker run --rm --volume $(pwd):/usr/local/src:rw \
-    --env MY_VALUE1="Hello World 1" \
-    --env MY_VALUE2="Hello World 2" \
-    appwrite/env-node-15.5:1.0.0 \
-    node index.js</code></pre>
-        </div>
-    </li>
-    <li>
-        <h3>CMD</h3>
-
-        <div class="ide margin-bottom" data-lang="bash" data-lang-label="CMD">
-            <pre class="line-numbers"><code class="prism language-bash" data-prism>docker run --rm --volume "%cd%":/usr/local/src:rw ^
-    --env MY_VALUE1="Hello World 1" ^
-    --env MY_VALUE2="Hello World 2" ^
-    appwrite/env-node-15.5:1.0.0 ^
-    node index.js"</code></pre>
-        </div>
-    </li>
-    
-    <li>
-        <h3>PowerShell</h3>
-
-        <div class="ide margin-bottom" data-lang="powershell" data-lang-label="PowerShell">
-            <pre class="line-numbers"><code class="prism language-powershell" data-prism>docker run --rm --volume ${pwd}:/usr/local/src:rw ,
-    --env MY_VALUE1="Hello World 1" ,
-    --env MY_VALUE2="Hello World 2" ,
-    appwrite/env-node-15.5:1.0.0 ,
-    node index.js"</code></pre>
-        </div>
-    </li>
-</ul>
-
-<p>The example above shows how you can mount your current working dir into the "appwrite/env-node-15.5:1.0.0" Cloud Functions runtime and execute a custom command to simulate exactly how the Appwrite server will execute the function on the Appwrite server. You can also pass multiple environment variables that your code might require.</p>
-
-<p>When running a Cloud Functions runtime for a coding language that uses a central directory for managing all dependencies, you will need to package all your dependencies in your working directory under the ".appwrite" folder as mentioned in the "Handling Dependencies" step above. This will ensure your function container can access all required packages and execute your function successfully.</p>
-
 <h2><a href="/docs/functions#demosExamples" id="demosExamples">Demos & Examples</a></h2>
 
```

Comment:
Did we mention this repo in the docs? https://github.com/appwrite/functions-starter
If not, should we?